server/src/handlers/chunk_handler.rs

```diff
@@ -865,8 +865,8 @@ pub struct SearchChunkData {
     pub get_total_pages: Option<bool>,
     /// Filters is a JSON object which can be used to filter chunks. This is useful for when you want to filter chunks by arbitrary metadata. Unlike with tag filtering, there is a performance hit for filtering on metadata.
     pub filters: Option<ChunkFilter>,
-    /// Set date_bias to true to bias search results towards more recent chunks. This will work best in hybrid search mode.
-    pub date_bias: Option<bool>,
+    /// Recency Bias lets you determine how much of an effect the recency of chunks will have on the search results. If not specified, this defaults to 0.0.
```

Comment:
Can you specify a range of what values work well here? 

I.e. `We recommend starting with x for a gentle boost and y for roughly a recency rerank`